build.ts

```diff
@@ -1,147 +1,86 @@
+#!/usr/bin/env -S deno run --allow-write
+import {
+  Colors,
+  flavorEntries,
+  FlavorName,
+  flavors,
+} from "https://deno.land/x/catppuccin@v1.1.1/mod.ts";
+import plist from "https://esm.sh/v135/plist@3.1.0";
 
-const template = `<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-  {{#each color}}
-  <key>{{key}}</key>
-  <dict>
-    <key>Color Space</key>
-    <string>sRGB</string>
-    <key>Red Component</key>
-    <real>{{col.red}}</real>
-    <key>Green Component</key>
-    <real>{{col.green}}</real>
-    <key>Blue Component</key>
-    <real>{{col.blue}}</real>
-    <key>Alpha Component</key>
-    <real>{{col.alpha}}</real>
-  </dict>
-  {{/each}}
-</dict>
-</plist>`;
+type ITermColor = {
+  "Red Component": number;
+  "Green Component": number;
+  "Blue Component": number;
+  "Alpha Component": number;
+  "Color Space": "sRGB";
+};
 
-for (let [flavour, colors] of Object.entries(variants)) {
-  const isLatte = flavour === "latte";
+const termcolors = (flavorName: FlavorName, suffix?: "Dark" | "Light") => {
+  const { dark, ...flavor } = flavors[flavorName];
 
-  colors = Object.fromEntries(
-    Object.entries(colors).map(([key, value]) => {
-      const [red, green, blue] = [
-        value.hex.slice(1, 3),
-        value.hex.slice(3, 5),
-        value.hex.slice(5, 7),
-      ].map((v) => parseInt(v, 16) / 255);
-      return [key, { red, green, blue, alpha: 1 }];
+  const colors = flavor.colorEntries.reduce((acc, [colorName, color]) => {
+    // iTerm needs rgb colors between 0.0 and 1.0
+    const [red, green, blue] = [color.rgb.r, color.rgb.g, color.rgb.b]
+      .map((v) => v / 255);
+
+    return {
+      [colorName]: {
+        "Red Component": red,
+        "Green Component": green,
+        "Blue Component": blue,
+        "Alpha Component": 1.0,
+        "Color Space": "sRGB",
+      },
+      ...acc,
+    };
+  }, {} as Colors<ITermColor>);
+
+  const mappings = {
+    "Ansi 0 Color": dark ? colors.surface1 : colors.subtext1,
+    "Ansi 1 Color": colors.red,
+    "Ansi 2 Color": colors.green,
+    "Ansi 3 Color": colors.yellow,
+    "Ansi 4 Color": colors.blue,
+    "Ansi 5 Color": colors.pink,
+    "Ansi 6 Color": colors.teal,
+    "Ansi 7 Color": dark ? colors.subtext1 : colors.surface2,
+    "Ansi 8 Color": dark ? colors.surface2 : colors.subtext0,
+    "Ansi 9 Color": colors.red,
+    "Ansi 10 Color": colors.green,
+    "Ansi 11 Color": colors.yellow,
+    "Ansi 12 Color": colors.blue,
+    "Ansi 13 Color": colors.pink,
+    "Ansi 14 Color": colors.teal,
+    "Ansi 15 Color": dark ? colors.surface1 : colors.subtext0,
+    "Background Color": colors.base,
+    "Foreground Color": colors.text,
+    "Link Color": colors.sky,
+    "Bold Color": colors.text,
+    "Cursor Color": colors.rosewater,
+    "Cursor Text Color": colors.base,
+    "Cursor Guide Color": { ...colors.text, alpha: 0.07 },
```

Comment:
This needed to be `"Alpha Component": 0.07`. https://github.com/catppuccin/iterm/blob/7a71cf49c88f9d729ffebeec9ddb9e83b09a0d00/colors/catppuccin-mocha.itermcolors#L39-L44